cmd/config.go

```diff
@@ -417,3 +419,110 @@ func overWriteConfig(cmd *cobra.Command, cfg *Config) error {
 	}
 	return err
 }
+
+// ValidatePath checks that a path is valid
+func (c *Config) ValidatePath(p *relayer.Path) (err error) {
+	if err = c.ValidatePathEnd(p.Src); err != nil {
+		return err
+	}
+	if p.Src.Version == "" {
+		return fmt.Errorf("source must specify a version")
+	}
```

Comment:
Let's re-order this check
```suggestion
	if p.Src.Version == "" {
		return fmt.Errorf("source must specify a version")
	}
	if err = c.ValidatePathEnd(p.Src); err != nil {
		return err
	}
```